pySDC/core/Sweeper.py

```diff
@@ -270,40 +280,96 @@ def rho(x):
             self.parallelizable = True
 
         elif qd_type == "MIN-SR-S":
+
             M = QDmat.shape[0] - 1
-            Q = coll.Qmat[1:, 1:]
-            nodes = coll.nodes
+            quadType = coll.quad_type
+            nodeType = coll.node_type
+
+            # Main function to compute coefficients
+            def computeCoeffs(m, a=None, b=None):
+                coll = CollBase(num_nodes=m, node_type=nodeType, quad_type=quadType)
+
+                Q = coll.Qmat[1:, 1:]
+                nodes = coll.nodes
```

Comment:
same comment as with `coll`, I think. Not super sure, honestly.

Reply:
Switched to `QM` notation